datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -370,6 +373,34 @@ func (f *ChainDataFetcher) handleRequestByType(reqType cfTypes.RequestType, shou
 		f.updateCheckpoint(ev.Block.Number().Int64())
 	}
 	handledBlockNumberGauge.Update(ev.Block.Number().Int64())
+	return nil
+}
+
+func (f *ChainDataFetcher) resetChainCh() {
+	for {
+		select {
+		case <-f.chainCh:
+		default:
+			return
+		}
+	}
+}
+
+func (f *ChainDataFetcher) resetRequestCh() {
+	for {
+		select {
+		case <-f.reqCh:
+		default:
+			return
+		}
+	}
+}
+
+func (f *ChainDataFetcher) forceStop() {
```

Comment:
What would be the difference between `forceStop` and `Stop`? Why do we need another stop function?

Reply:
`Stop` stops chaindatafetcher. I want to make only fetching/rangeFetching jobs to be stopped and clear the channels.

Reply:
In that purpose, `pause` would be more appropriate. `Stop` terminates ChainDataFetcher, but `forceStop` doesn't. `forceStop` sounds like it will `Stop` anyway though it might be ungraceful.

Reply:
I renamed `forceStop` to `pause`. Thanks.